### ISSUE TITLE
Field size in dbml file

### ISSUE THREAD

**Issue by akuiteo-dbdocs** [holistics/dbml#644]
Before using the automatic generation of the DBML file for SQL server, I had queries allowing me to generate my file in DBML format.
In these scripts, I retrieved the field type and size, which was very useful when viewing fields in dbdocs. 

Example : 
```
Table "dbo"."B_BATCH_PLANIF" {
  "BBPL_ANNEE" numeric(14,2)
  "BBPL_AV_ANNEE" nvarchar(60) [note: 'Mode CRON : Annee(s)']
  "BBPL_AV_HEURE" nvarchar(10) [note: 'Mode CRON : Heures']
}
```

With “dbdocs db2dbml mssql” I no longer have this information, which is a big loss compared with what I used to produce. Would it be possible to have this data? 

```
Table "dbo"."B_BATCH_PLANIF" {
  "BBPL_ANNEE" numeric
  "BBPL_AV_ANNEE" nvarchar [note: 'Mode CRON : Annee(s)']
  "BBPL_AV_HEURE" nvarchar [note: 'Mode CRON : Heures']
}
```

**Comment by huyleminh01** [comment]
Hi @akuiteo-dbdocs,

Thank you for reporting the bug to us.

It maybe a bug related to db2dbml command, we will investigate this issue and get back to you later.

Thanks.

**Comment by huyleminh01** [comment]
Hi @akuiteo-dbdocs ,

We checked and confirmed that our connector does not fetch the field size _(numeric scale, numeric precision, and string length)_.

So we have released a fixed version in `@dbml/connector@3.9.1`. We will update it to dbdocs cli later. But you can try [dbml cli ](https://dbml.dbdiagram.io/cli#generate-dbml-directly-from-a-database)to test our new fix first.

Thanks.

**Comment by huyleminh01** [comment]
Hi @akuiteo-dbdocs ,

There is good news: we've just released [`dbdocs cli v0.13.1`](https://www.npmjs.com/package/dbdocs) contains this fix 🥳 . Please try it out

**Comment by akuiteo-dbdocs** [comment]
It's work like a charm ! Thanks a lot
